cyberdrop_dl/crawlers/luxuretv.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from cyberdrop_dl.crawlers.crawler import Crawler, SupportedPaths
+from cyberdrop_dl.data_structures.url_objects import AbsoluteHttpURL
+from cyberdrop_dl.utils import css
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+class Selector:
+    VIDEO_PLAYER = "video.video-js > source"
+    TITLE = "h1.title-right"
+    NEXT_PAGE = "div#pagination > a:-soup-contains('»')"
+    VIDEOS_THUMBS = "div.contents div.content a"
+
+
+PRIMARY_URL = AbsoluteHttpURL("https://luxuretv.com")
+
+
+class LuxureTVCrawler(Crawler):
+    SUPPORTED_PATHS: ClassVar[SupportedPaths] = {
+        "Video": "/videos/<name>-<id>.html",
+        "Search": "/searchgate/videos/<search>/...",
+    }
+    DOMAIN: ClassVar[str] = "luxuretv"
+    FOLDER_DOMAIN: ClassVar[str] = "LuxureTV"
+    DEFAULT_TRIM_URLS: ClassVar[bool] = False
+    PRIMARY_URL: ClassVar[AbsoluteHttpURL] = PRIMARY_URL
+    NEXT_PAGE_SELECTOR: ClassVar[str] = Selector.NEXT_PAGE
+    _RATE_LIMIT = 3, 10
+
+    async def fetch(self, scrape_item: ScrapeItem) -> None:
+        if "searchgate" in scrape_item.url.parts:
+            return await self.search(scrape_item)
+        elif "videos" in scrape_item.url.parts:
+            video_id = scrape_item.url.name.split("-")[-1].split(".")[0]
+            return await self.video(scrape_item, video_id)
+        raise ValueError
+
+    @error_handling_wrapper
+    async def search(self, scrape_item: ScrapeItem) -> None:
+        title: str = ""
+        url = scrape_item.url
+        if url.name and not url.name.endswith(".html"):
+            url = url / ""
+        async for soup in self.web_pager(url, cffi=True):
+            if not title:
+                title = css.select_one_get_text(soup, Selector.TITLE)
+                title = title.split(":")[-1].strip()
+                title = self.create_title(f"{title} - [search]")
+                scrape_item.setup_as_album(title)
```

Comment:
We can use the query bypass that and also skip the selector as well

```suggestion
    async def search(self, scrape_item: ScrapeItem, query: str) -> None:
        title = query.replace(".", " ").capitalize()
        title = self.create_title(f"{title} [search]")
        scrape_item.setup_as_album(title)
        url = scrape_item.url
        if url.name and not url.name.endswith(".html"):
            url = url / ""
            
        async for soup in self.web_pager(url, cffi=True):        
```